pstatus-notifications-workflow-ktor/src/main/kotlin/HealthCheckTemporalServer.kt

```diff
@@ -8,18 +8,21 @@ import io.temporal.serviceclient.WorkflowServiceStubsOptions
 
 /**
  * health check implementation of Temporal
+ * @param temporalTarget String
  */
 
-class HealthCheckTemporalServer : HealthCheckSystem("Temporal Server") {
+class HealthCheckTemporalServer(temporalTarget: String) : HealthCheckSystem("Temporal Server") {
```

Comment:
I suggest passing in the entire `TemporalConfig` object config here, in case other params are eventually used it makes for fewer changes and simpler c'tor.